rag/README.md

```diff
@@ -0,0 +1,76 @@
+# Steps to create the RAG tool
+
+This readme file documents two steps:
+1) Deploy a standalone milvus container and initiate it with company documents
+2) Deploy a RAG API service that can be used as a tool by the Agent
+
+## 1. [Milvus](https://milvus.io/docs/) Deployment and [pymilvus](https://pymilvus.readthedocs.io/) SDK Usage
+
+This guide explains how to deploy Milvus using Kubernetes (specifically OpenShift), connect to it using the `pymilvus` SDK, and run operations on Milvus through a Jupyter Notebook.
+
+### Prerequisites
+
+1. **Kubernetes Cluster**: Ensure you have access to a Kubernetes cluster (e.g., OpenShift).
+2. **kubectl/oc CLI**: Command-line interface to manage Kubernetes/OpenShift resources.
+
+### Deployment Steps
+
+#### 1. Apply Milvus Deployment YAML with the following command:
+
+   ```bash
+   kubectl apply -f vector_db/standalone_milvus.yaml
+   ```
+
+#### 2. Connecting to Milvus with pymilvus
+
+#### Example Notebook Code to Delete a Collection
+
+1. Open Jupyter Notebook `vector_db/vector_db_insert.ipynb` and add the Milvus host from step 1.
+
+2. Run all the notebook cells
+
+## RAG tool: Uvicorn FastAPI Deployment on OpenShift
+
+This guide provides step-by-step instructions to build a container image for a `uvicorn` FastAPI application, push it to a container registry (Quay.io), and deploy it on OpenShift.
```

Comment:
would be more clear to phrase it as "This guide provides step-by-step instructions to build a container image for a `uvicorn` FastAPI application to expose the RAG service, push it to a container registry (Quay.io) and deploy it on OpenShift."